### DISCUSSION TITLE
Lints with no configuration

### DISCUSSION THREAD

**Discussion by BigfootN** [rust-lang/rust-clippy#8533] [Q&A] [answered]
Hello,

I noticed that some lints (ex. `absurd_extreme_comparisons`) do not have any `Configuration` section.

Does this mean that:
- They can only be controlled in the code ? (not in the `clippy.toml` file)
- Or that they can only be controlled by their group ?

Thank you very much in advance

**Comment by flip1995** [comment, marked as answer]
Note that `Configuration` in Clippy means configuring the lint and not the _lint level_. The lint level can only be configured in the code or the command line. Discussion about a configuration file for lint levels exists, see https://github.com/rust-lang/cargo/issues/5034.

The configuration that can be done in the `clippy.toml` file only makes sense for some lints. A good example is the `too_many_arguments` lint, where you can specify in the `clippy.toml` file how many arguments you want to allow. 

If you find a lint that could benefit from a configuration option, feel free to open an issue about it or ask over on Zulip.

**Reply by BigfootN** [reply]
Thank you very much for your quick and greatly detailed answer.
